save/10981.0.txt

```diff
@@ -0,0 +1,43 @@
+@0
+
+
+
+@1
+MOV UP, ACC
+
+#moves the value to the accoumlator for us to be able to add them together
+ADD ACC
+
+MOV ACC, RIGHT
+
+#moves the accoumlator afer adding it together
```

Comment:
Comment isn't very clear here. What does "adding it together" mean in this context?